ui/src/main/js/Navigation.js

```diff
@@ -56,6 +60,16 @@ class Navigation extends Component {
         );
     }
 
+    createEmailNavItem(uriPrefix) {
```

Comment:
Can this and `createSlackNavItem` be abstracted into one method that also takes an "info" param?